test/boost/cql_query_test.cc

```diff
@@ -5218,8 +5218,16 @@ SEASTAR_TEST_CASE(timeuuid_fcts_prepared_re_evaluation) {
     });
 }
 
+template<typename Func>
+static future<> with_parallelized_aggregation_enabled(Func&& func) {
```

Comment:
Please use std::function to reduce template proliferation.

Reply:
and add _thread to the name, to reduce the chance of confusion.

Reply:
changed

Reply:
> and add _thread to the name, to reduce the chance of confusion.

You forgot this

Reply:
The page didn't refresh, updated